.eslintrc.json

```diff
@@ -67,6 +81,44 @@
       "parserOptions": {
         "project": "./tsconfig.json"
       }
+    },
+    {
+      "files": ["./**/*.{js,jsx,ts,tsx}"],
+      "rules": {
+        "import/order": [
+          "error",
+          {
+            "alphabetize": { "order": "asc", "caseInsensitive": true },
+            "groups": [
+              "builtin",
+              "external",
+              "internal",
+              "parent",
+              "sibling",
+              "index",
+              "object",
+              "type"
+            ],
+            "newlines-between": "never",
```

Comment:
How would it be like if we make this "always"? Something too weird?

Reply:
ah, will it require newlines between `import { foo } from './foo'` and `import { bar } from './bar'`?

Reply:
> How would it be like if we make this "always"? Something too weird?

If set to always, at least one new line between each group will be enforced, and new lines inside a group will be forbidden.

```ts
/* eslint import/order: ["error", {"newlines-between": "always"}] */
import fs from 'fs';
import path from 'path';

import index from './';

import sibling from './foo';
```

> ah, will it require newlines between import { foo } from './foo' and import { bar } from './bar'?

The following is only a personal opinion:
1. Now everyone is using auto import
2. Configuring eslint rules will help us sort  imports

In summary, there are now very few cases where the imports section needs to be managed manually, then it is not so necessary to add blank lines just for the sake of readability.

Reply:
Okay, it's just unfortunate for me that the rule doesn't allow adding one line only between absolute path imports and relative path imports.

Reply:
I will study this again, and if it can be realized, I’ll launch a PR.